README.md

```diff
@@ -1,5 +1,15 @@
-Handbook for writing clean, performant Lua code for GTA 5's FiveM framework.
+## 📖 Effective FiveM Lua
+A handbook of tips and guidelines for writing clean, effective code in Lua for GTA 5's FiveM Modding Framework
 
-https://manason.github.io/effective-fivem-lua
+## Table of Contents
+1. [Foreword](https://manason.github.io/effective-fivem-lua/#foreword)
+2. [Structure/Scope](https://manason.github.io/effective-fivem-lua/structure/#structurescope)
+3. [Functions](https://manason.github.io/effective-fivem-lua/functions/#functions)
+4. [Events](https://manason.github.io/effective-fivem-lua/events/#events)
+5. [Tables](https://manason.github.io/effective-fivem-lua/tables/#tables)
+6. [Variables](https://manason.github.io/effective-fivem-lua/variables/#variables)
+7. [Conditionals](https://manason.github.io/effective-fivem-lua/conditionals/#conditionals)
+8. [Natives](https://manason.github.io/effective-fivem-lua/natives/#natives)
```

Comment:
I'd rather not maintain an extra table of contents in the README as the website already has the table of contents.